ctl/import.go

```diff
@@ -108,10 +105,14 @@ func (cmd *ImportCommand) Run(ctx context.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "getting schema")
 	}
+
+	var useColumnKeys, useRowKeys bool
 	for _, index := range schema {
 		if index.Name == cmd.Index {
+			useColumnKeys = index.Options.Keys
 			for _, field := range index.Fields {
 				if field.Name == cmd.Field {
+					useRowKeys = field.Options.Keys
 					fieldType = field.Options.Type
 				}
```

Comment:
Similary, a `break` after this line would be more efficient to cut the loop early.

Reply:
Github moved the location of the line I've made the comment. It's right after the closing brace of the `for _, field ... ` loop.